packages/app/src/app/overmind/effects/vscode/ModelsHandler.ts

```diff
@@ -161,7 +172,7 @@ export class ModelsHandler {
     // promise, because there might be multiple operations fired before
```

Comment:
I'm wondering with this comment, wouldn't the operations be applied all on the same version of code? Not 100% sure.